bokehjs/src/compiler/prelude.ts

```diff
@@ -1,5 +1,34 @@
-//import license from "../../../LICENSE.txt"
-const license = "TODO"
+// TODO import license from "../../../LICENSE.txt"
+const license = `\
+Copyright (c) 2012 - 2019, Anaconda, Inc., and Bokeh Contributors
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+Redistributions of source code must retain the above copyright notice,
+this list of conditions and the following disclaimer.
+
+Redistributions in binary form must reproduce the above copyright notice,
+this list of conditions and the following disclaimer in the documentation
+and/or other materials provided with the distribution.
+
+Neither the name of Anaconda nor the names of any contributors
+may be used to endorse or promote products derived from this software
+without specific prior written permission.
```

Comment:
We should open a discussion with @pzwang about removing the Anaconda mentions and just having verbatim BSD 30-clause text everywhere